agent/src/main/java/com/microsoft/hydralab/agent/runner/appium/AppiumListener.java

```diff
@@ -31,6 +32,7 @@ public class AppiumListener extends RunListener {
     private final AnimatedGifEncoder e = new AnimatedGifEncoder();
     private final String pkgName;
     DeviceManager deviceManager;
+    IPerformanceListener performanceListener;
```

Comment:
Do we need package visibility here? Or add `private` with this field.

Reply:
Currently, there is no need for package visibility. I add `private final` to the field. Thanks.